apps/website/src/routes/website/_webview/editor/caret.ts

```
@@ -0,0 +1,107 @@
+      }
+      return false;
+    } else {
+      const $head = sel.$head;
+      const node = $head.textOffset ? null : dir < 0 ? $head.nodeBefore : $head.nodeAfter;
+
+      if (!node || node.isText) return false;
+
+      const nodePos = dir < 0 ? $head.pos - node.nodeSize : $head.pos;
+
+      if (node.isAtom || !node.isLeaf) {
+        if (NodeSelection.isSelectable && NodeSelection.isSelectable(node)) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)